src/Build/Evaluation/Expander.cs

```diff
@@ -2068,12 +2068,12 @@ ExpanderOptions options
                             return false;
                         }
                         int currentLength = builder.Length - startLength;
-                        if (currentLength + item.Key.Length > CharacterLimitPerExpansion)
+                        if (item.Key is string keyToTruncate && currentLength + keyToTruncate.Length > CharacterLimitPerExpansion)
```

Comment:
if `item.Key` is an empty string, would the condition still succeed? I assume the reason for doing so is to avoid empty values.

Reply:
I answered my own question: https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYCMBYAUKgZgAIMiBhIgbzyNpPXSprpYGVgwBLKAcyIGciAXiIAiUQG5mLWtJmcAZgApBnQey68iPAJRyW1XDJapMATiWiAKgFN+wUTol19dAL5yPuN0A==

It does. So this doesn't actually change functionality if `item.Key` is empty right?

Reply:
It will and that's actually a nice readability simplification too.